shared/react-native/android/app/src/main/java/io/keybase/ossifrage/KeyStore.java

```diff
@@ -62,91 +64,140 @@ private String keyStoreAlias(final String serviceName) {
     @SuppressLint("CommitPrefEdits")
     @Override
     public void clearSecret(final String serviceName, final String key) throws Exception {
-        prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
+        String id = serviceName + ":" + key;
+        NativeLogger.info("KeyStore: clearing secret for " + id);
+
+        try {
+            prefs.edit().remove(sharedPrefKeyPrefix(serviceName) + key).commit();
```

Comment:
here and below, I didn't actually change any code, just re-indented them for the try blocks